### DISCUSSION TITLE
Help with a refund.

### DISCUSSION THREAD

**Discussion by sebaslml** [Security-Onion-Solutions/securityonion#6059] [Unsupported Versions] [answered]
Hi, I need a refund from Security Onion because I am not happy with the free trial from AWS using security onion 2 and they added some charges about EBS storage, and they tell me to ask for a refund from Security Onion. I need someone to help me to eliminate this charges.

**Comment by dougburks** [comment, marked as answer]
From the Free Trial section of https://securityonion.net/aws:

> Try one unit of this product for 30 days. There will be no software charges for that unit, but AWS infrastructure charges still apply.

**Reply by sebaslml** [reply]
The product's Free Trial limit was accidentally exceeded, I cancel my trial to avoid more charges, but I need this refund.

**Reply by dougburks** [reply]
We don't normally grant refunds for users exceeding their trial limits. You can use the email link at https://securityonion.net to send your trial account info and we will investigate.

**Reply by sebaslml** [reply]
I will send an e-mail. Thanks for helping me.!
